package.json

```
@@ -43,24 +43,27 @@
     "url": "https://github.com/uphold/koa-error-mapper.git"
   },
   "scripts": {
-    "changelog": "github-changelog-generator --owner=uphold --repo=koa-error-mapper --future-release=v$npm_package_version > CHANGELOG.md",
-    "lint": "eslint index.js mappers test",
-    "test": "jest --coverage",
-    "version": "npm run changelog && git add -A CHANGELOG.md"
+    "changelog": "uphold-scripts changelog $npm_package_version",
```

```suggestion
    "changelog": "uphold-scripts changelog v$npm_package_version",
```

Because this project is prepending `v` to the version numbers in its releases.